x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/install_kibana_assets_button.tsx

```diff
@@ -54,23 +54,11 @@ export function InstallKibanaAssetsButton({
       isLoading={isLoading}
       onClick={handleClickInstall}
     >
-      {isLoading ? (
-        <FormattedMessage
-          id="xpack.fleet.integrations.installPackage.kibanaAssetsInstallingButtonLabel"
-          defaultMessage="Installing Kibana assets in current space"
-          values={{
-            title,
-          }}
-        />
-      ) : (
-        <FormattedMessage
-          id="xpack.fleet.integrations.installPackage.kibanaAssetsInstallButtonLabel"
-          defaultMessage="Install Kibana assets in current space"
-          values={{
-            title,
-          }}
-        />
-      )}
+      <FormattedMessage
+        id="xpack.fleet.integrations.installPackage.kibanaAssetsInstallingButtonLabel"
+        defaultMessage="{isLoading, select, true {Installing} other {Insall}} Kibana assets in current space"
```

Comment:
Fleet code was already approved, however I just noticed a small typo here: `Insall` instead of `Install`